OVS_Bonding/README.md

```diff
@@ -0,0 +1,55 @@
+# <u>Setting up network bonding on specific interfaces during the ignition process</u>
+
+## Overview 
+This guide explains how to bond specific NICs during the initial OpenShift installation with OVS and supplies the Ignition and scripts to achieve this.
+
+## Important
+- All raw files and scripts are located in the RawFiles folder.
+- This guide describes the customisations needed for installation when using a PXE server.
+<br>
+
+## Prerequisites
+- PXE server with ability to set kernel arguments.
+- Web accessible location(HTTP only) to upload the <i> capturemacs.ign </i> file .
```

Comment:
you can do italic formatting using astrisk:  *example*. 
you can advise the github formatting [guide](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax).
```suggestion
- Web accessible location (HTTP only) to upload the *capturemacs.ign* file .
```